package.json

```diff
@@ -141,5 +143,16 @@
       "@commitlint/config-conventional"
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
Is this needed seeing the content of `commit-msg`? 🤔

Reply:
Or could the change to commit-msg be removed? I feel like one or the other is superfluous (and running it on commit-msg feels a bit wrong compared to pre-commit 🤔 )

Reply:
does not work without `commit-msg` :( . So keeping it there